### PR TITLE
Add prelim motor spec scripts

### PR DESCRIPTION
**Pull request by matthew-reynolds**

### CODE REVIEW

File: matlab/speed_spec.m [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+gait_hip_swing_angle = deg2rad(3.5) + deg2rad(30); % unrealistic but large values yield less efficient steps so it just goes into fs
+
+effective_leg_length = sqrt(upper_leg_length^2 + lower_leg_length^2 -2*upper_leg_length*lower_leg_length*cos(deg2rad(180)-gait_knee_angle));
+
+% HUGE simplifying assumptions here. ignores that linear speed of step
+% isn't linearly proportional to angular vel... amoung many other
+% assumptions
+effective_step_length = effective_leg_length*sin(gait_hip_swing_angle);
+
+% more OMEGA BIG simplifications
```
Comment (wmmc88) [inline matlab/speed_spec.m:18]: nice

File: matlab/torque_calc_old.m [hunk 2]
```diff
@@ -0,0 +1,17 @@
+l_u = 0.4/2;
```
Comment (wmmc88) [inline matlab/torque_calc_old.m]: do we still need this?

**Review by wmmc88 — DISMISSED** [review]

File: matlab/torque_calc_old.m (hunk 2 above)
Comment (matthew-reynolds) [reply matlab/torque_calc_old.m]: doubt it. Follow-up lol

Comment (matthew-reynolds) [reply matlab/torque_calc_old.m]: jk done in 3981e1b

**Review by wmmc88 — APPROVED** [review]

**Pull request merged**